examples/reference/widgets/TextInput.ipynb

```diff
@@ -63,7 +63,7 @@
    "source": [
     "### Basic Usage\n",
     "\n",
-    "The `TextInput` widget provides a simple text entry form and supports a `placeholder` which is displayed when the `value` is empty:"
+    "The `TextInput` widget provides a simple text entry form:"
    ]
```

Comment:
Its simply too hard to get "form validation".

I would expect it to be almost automatic to get `error_state` set to `True` and the label or tooltip to update to something useful here.

```bash
import panel as pn
import panel_material_ui as pmui
import re

pn.extension()

def validation_func(value):
    if not value or not value.strip():
        raise ValueError("Email cannot be empty")
    if not re.match(r"^[\w\.-]+@[\w\.-]+\.\w+$", value):
        raise ValueError("Invalid email format")

widget = pmui.widgets.TextInput(value="", placeholder="Enter your email", label="User Email")

pn.bind(validation_func, widget, watch=True)

widget.servable()
```